src/Helpers.py

```
@@ -1,6 +1,6 @@
 import random
 from enum import Enum
-
+from geopy import distance
```

Ruff (0.8.2)

`geopy.distance` imported but unused

Remove unused import: `geopy.distance`

(F401)